app/models/viewmodels/ilequery/IleQueryConverter.scala

```diff
@@ -41,6 +43,9 @@ class IleCodeMapper @Inject()(decoder: Decoder) {
   def statusOfEntryMucr(code: String)(implicit messages: Messages): Content =
     decoder.mucrSoe(code).map(soeCode => HtmlContent(htmlString(soeCode))).getOrElse(Empty)
 
-  def statusOfEntryCombined(code: String)(implicit messages: Messages): Content =
-    decoder.soeCombined(code).map(soeCode => HtmlContent(htmlString(soeCode))).getOrElse(Empty)
+  def statusOfEntryAll(code: String)(implicit messages: Messages): Content =
+    decoder.allSoe(code).map(soeCode => HtmlContent(htmlString(soeCode))).getOrElse(Empty)
+
+  def transport(transport: Transport): Content =
+    Text((transport.transportId.toSeq ++ transport.nationality.map(countryName).toSeq).mkString(", "))
```

Comment:
I've just checked and you don't need `.toSeq` calls. Can do just `Text((transport.transportId ++ transport.nationality.map(countryName)).mkString(", "))`

Reply:
You are correct.  Changed and created test for it